### DISCUSSION TITLE
option to skip only cloud cache

### DISCUSSION THREAD

**Discussion by skoppell** [nrwl/nx#26895] [Questions]
Hi,

Currently we have --skip-nx-cahe which ignores both local and cloud cache.  Is there a way to configure the commands to look for for only local cache if available otherwise skip looking for cloud cache.

We have cloud-cache enabled for our project. (Nx+Angular). In our regular pull-request pipelines, I have configured the cache inputs such that the affected command results in minimal projects and which leads to test target being run only for those minimal set of projects. however, to make sure all the test in the repo are in good state, I want to configure a scheduled pipeline to run the test target on all projects once in a day. So, i configured it using run-many --all with --skip-nx-cache.

Our monorepo has more than 200 libraries so if the job fails due to a timeout, i would like to re-run the job. but that would result in running the tests on all project again as it uses --skip-nx-cache argument. If there was a option to skip only cloud cache, It would help the subsequent run to pick the results from local cache and run the task only for the projects failed in the previous run. (as the job will be run on the same CI node in the subsequent run, local cache will be available on that node.)

Or Do we have a way to achieve what i am looking for ?

**Comment by robsonos** [comment]
Hi @skoppell, I have just created [this GitHub action](https://github.com/marketplace/actions/nx-cloud-skipper) to solve a similar issue. Not sure if it will solve your issue though. Cheers
